### PR TITLE
Unsub

### PR DESCRIPTION
**Pull request by chengkangzai**

### CODE REVIEW

**Comment by nubpro** [comment]
How to test this feature ah?

**Comment by chengkangzai** [comment]
> How to test this feature ah?

tbh I have no idea how to test it but that's what Julius told so ...

**Comment by chengkangzai** [comment]
@nubpro i did my testing ady

File: components/header.tsx [hunk 1]
```diff
@@ -10,12 +10,13 @@ export default function Header() {
 
     useEffect(() => {
         // TODO: Move this outside of header, probably in _app.tsx (need to create an AuthProvider I think)
-        const publicRoutes = ['/', '/login', '/resetpassword'];
+        const publicRoutes = ['/', '/login', '/resetpassword', '/faq', '/r'];
```
Comment (nubpro) [inline components/header.tsx]: /faq is a not available route right?

**Review by nubpro — CHANGES_REQUESTED** [review]

File: components/header.tsx (hunk 1 above)
Comment (chengkangzai) [reply components/header.tsx]: oppsie

Comment (chengkangzai) [reply components/header.tsx]: Resolve in https://github.com/nubpro/nakvaksin/pull/27/commits/7d7c464b4045a737f2794ec9f7f37dbed8aa6649

**Review by nubpro — APPROVED** [review]
Are we ready to ship this?

**Comment by chengkangzai** [comment]
To the moon!

**Pull request merged**